00-Login-Kt/app/src/main/res/layout/activity_main.xml

```diff
@@ -36,13 +46,60 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <TextView
-        android:id="@+id/userProfile"
+    <LinearLayout
+        android:id="@+id/metadata_panel"
+        android:visibility="invisible"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:gravity="center"
+        android:layout_marginTop="16dp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/button_logout"
-        android:layout_margin="36dp"/>
+        app:layout_constraintTop_toBottomOf="@id/user_profile">
+
+        <TextView
+            android:id="@+id/text_heading_metadata"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textStyle="bold"
+            android:text="Metadata"/>
+
+        <TextView
+            android:id="@+id/text_info_metadata"
+            android:layout_width="match_parent"
+            android:layout_height="50dp"
+            android:layout_margin="8dp"
+            android:textAlignment="center"
+            android:text="Use the buttons and text box below to patch the 'country' field from 'user_metadata'"
```

Comment:
It's just a sample app but would we be better off putting these into string resources as good practise?

Reply:
for these sample purposes, I wouldn't use string resources. Those are meant to be used when you plan to reuse them across the app and provide translations.